scripts/UpdateBuildTools.ps1

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env powershell
+
+<#
+.SYNOPSIS
+    Updates the build tools version and generates a commit message with the list of changes
+.PARAMETER RepoRoot
+    The directory containing the repo
+.PARAMETER GitCommitArgs
+    Additional arguments to pass into git-commit
+.PARAMETER NoCommit
+    Make changes without executing git-commit
+.PARAMETER Force
+    Specified this to make a commit with any changes
+#>
+[cmdletbinding(SupportsShouldProcess = $true)]
+param(
+    [string]$RepoRoot,
+    [string[]]$GitCommitArgs = @(),
+    [switch]$NoCommit,
+    [switch]$Force
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 2
+
+if (-not $RepoRoot) {
+    $RepoRoot = Resolve-Path "$PSScriptRoot\.."
+}
+
+Import-Module "$PSScriptRoot/common.psm1" -Scope Local -Force
+
+function Get-KoreBuildVersion {
+    $lockFile = "$RepoRoot/korebuild-lock.txt"
+    if (!(Test-Path $lockFile)) {
+        return ''
+    }
+    $version = Get-Content $lockFile | Where-Object { $_ -like 'version:*' } | Select-Object -first 1
+    if (!$version) {
+        Write-Error "Failed to parse version from $lockFile. Expected a line that begins with 'version:'"
+    }
+    $version = $version.TrimStart('version:').Trim()
+    return $version
+}
+
+Push-Location $RepoRoot
+try {
+    Ensure-Git
+
+    $oldVersion = Get-KoreBuildVersion
+
+    & "$RepoRoot/run.ps1" -Update default-build '/t:Cow'
```

Comment:
Isn't running default-build with `/t:Cow` the same as running `install-tools`? If not I feel like it's worth clarifying what all we expect this to do.

Reply:
Yeah. I just need some kind of no-op command so the korebuild-lock.txt file gets updated. I'll add a comment.